internal/vppinit/apiparams/apiparams.go

```diff
@@ -0,0 +1,153 @@
+// Copyright (c) 2024 Nordix Foundation.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package apiparams provides parsing factilty for configuration parameters
+// file for vpp init
+package apiparams
```

Comment:
Hmm.. Seems to me we can move this file into vppinit package. 

@glazychev-art , @ljkiraly  Thoughts?

Reply:
Moved it to vppinit.